### ISSUE TITLE
Add some sort of labeling

### ISSUE THREAD

**Issue by McTwist** [McTwist/Support_EventScript#10]
To reduce the complexity of event numbering for some output events, implement some way to label each event or groups of events.

[Platypi]( https://forum.blockland.us/index.php?topic=314701.msg9713710#msg9713710) suggested C-like labeling with Python-like indexing. This would be totally possible and is a great addition to #5.

Quick Example:
```
onStart:
onRelay -> Self -> toggleEvent([onStart:onEnd])
onEnd:
```

**Comment by McTwist** [comment]
Indexing was added on commit 8d5b42f.

**Comment by McTwist** [comment]
Solved through bb18fc2.
